### PR TITLE
[Zeppelin-3224] - Fix "ultimate" visualization

### PR DESCRIPTION
**Pull request by Savalek**
### What is this PR for?
This PR fixes problems with incorrect data visualization using "ultimate". With certain aggregations, the data was not displayed. Also PR improves sorting by columns / rows, before PR it compared by Unicode encoding, but now if there are only numeric data on the axis, sorting will be by values.

### What type of PR is it?
[Bug Fix & Improvement]

### What is the Jira issue?
[ZEPPELIN-3224](https://issues.apache.org/jira/browse/ZEPPELIN-3224)

### Screenshots (if appropriate)
**Error:**

error

**Now:** 

default

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: zeppelin-web/src/app/tabledata/tabledata.js [hunk 1]
```diff
@@ -59,6 +59,9 @@ export default class TableData extends Dataset {
         if (i === 0) {
           columnNames.push({name: col, index: j, aggr: 'sum'});
         } else {
+          if (col.match('^[0-9]+([.,][0-9]+)?$')) {
```
Comment (felixcheung) [inline zeppelin-web/src/app/tabledata/tabledata.js]: negative number?

Comment (felixcheung) [reply zeppelin-web/src/app/tabledata/tabledata.js]: instead of applying a regex, is there a js function to try parse the string as float?

**Comment by Savalek** [comment]
@felixcheung 
I found a better solution for checking for a number.

File: zeppelin-web/src/app/tabledata/advanced-transformation-util.js [hunk 2]
```diff
@@ -1339,3 +1339,17 @@ export function fillDrillDownRow(schema, obj, rows, key,
...
 }
+
+export function sortWithNumberSupport(arr) {
+  let isNumeric = function(n) {
+    return !isNaN(parseFloat(n)) && isFinite(n);
+  };
+
+  if (arr.every(isNumeric)) {
+    return arr.sort(function(a, b) {
+      return parseFloat(a, 10) - parseFloat(b, 10);
```
Comment (Leemoonsoo) [inline zeppelin-web/src/app/tabledata/advanced-transformation-util.js]: Does `parseFloat` take second parameter?

**Comment by Savalek** [comment]
@Leemoonsoo 
I have confused the functional with parseInt().
I fixed it.

**Comment by Leemoonsoo** [comment]
Thanks @Savalek for great improvement. LGTM and merge to master and branch-0.8 if no further comment.

**Pull request closed without merging**